frontend/src/Trick.tsx

```diff
@@ -95,7 +96,9 @@ const Trick = (props: IProps) => {
             key={id}
             id={id}
             label={
-              namesById[id] + (id === props.landlord ? " (当庄)" : "") + suffix
+              namesById[id] +
```

Comment:
if `landlord_emoji` isn't set, we should still show (当庄) ideally?

Reply:
That is exactly what it is. I want to keep the default as (当庄) to make you happen :)

Reply:
oh -- is `landlord_emoji` really just `landlord_label` then? I was confused

Reply:
> > Probably want to invest in making the "initialize game" thing a bit more organized sometime soon; the number of settings is really stacking up.
> 
> I have more options in mind:) one of the reasons I asked about use bootstrap, actually

doing some more UI work here seems reasonable -- not sure about Bootstrap in particular since we really only have multiselects here. You could imagine using `react-select` for this.

I think the more important bit is going to be figuring out which of these settings can live in a settings pane vs. being available on the front page, since making configuration complicated is less good for players

Reply:
> oh -- is `landlord_emoji` really just `landlord_label` then? I was confused

no. The landlord label has player name as well. this emoji string is the tail part of the label, so the landlord label is <player name>' '(当庄) or <player name>' '<emoji>. the emoji string is just a utf8 string

Reply:
> > oh -- is `landlord_emoji` really just `landlord_label` then? I was confused
> 
> no. The landlord label has player name as well. this emoji string is the tail part of the label, so the landlord label is ' '(当庄) or ' '. the emoji string is just a utf8 string

so the landlord label is <<player_name>>' '(当庄) or <<player_name>>' '<<emoji>>. the emoji string is just a utf8 string

Reply:
or <<player_name>>' '<<emoji>>

Reply:
stupid! or <<player_name>>' '<<landlord_emoji>>